src/gql/queries/analytics/get-patch-filters-attributes.ts

```diff
@@ -0,0 +1,10 @@
+import gql from "graphql-tag";
+
+/* eslint-disable */
```

Comment:
can you disable a specific rule instead of everything in the file.

Reply:
individual disables for a line cannot be done in a string literal, so no

Reply:
I meant adding `graphql/template-strings` to eslint-disable